### PR TITLE
[Doc] Update bips.md for Segregated Witness

### PR DESCRIPTION
**Pull request by fanquake**

### CODE REVIEW

**Comment by btcdrak** [comment]
Should add BIP146 (NULLDUMMY)

**Comment by fanquake** [comment]
Updated with BIP147.

File: doc/bips.md [hunk 1]
```diff
@@ -26,4 +26,9 @@ BIPs that are implemented by Bitcoin Core (up-to-date up to **v0.13.0**):
...
 * [`BIP 130`](https://github.com/bitcoin/bips/blob/master/bip-0130.mediawiki): direct headers announcement is negotiated with peer versions `>=70012` as of **v0.12.0** ([PR 6494](https://github.com/bitcoin/bitcoin/pull/6494)).
 * [`BIP 133`](https://github.com/bitcoin/bips/blob/master/bip-0133.mediawiki): feefilter messages are respected and sent for peer versions `>=70013` as of **v0.13.0** ([PR 7542](https://github.com/bitcoin/bitcoin/pull/7542)).
+* [`BIP 141`](https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki): Segregated Witness (Consensus Layer) as of **v0.13.0** ([PR 7910](https://github.com/bitcoin/bitcoin/pull/7910)).
+* [`BIP 143`](https://github.com/bitcoin/bips/blob/master/bip-0143.mediawiki): Transaction Signature Verification for Version 0 Witness Program as of **v0.13.0** ([PR 7910](https://github.com/bitcoin/bitcoin/pull/7910)).
+* [`BIP 144`](https://github.com/bitcoin/bips/blob/master/bip-0144.mediawiki): Segregated Witness as of **0.13.0** ([PR 7910](https://github.com/bitcoin/bitcoin/pull/7910)).
```
Comment (MarcoFalke) [inline doc/bips.md]: The changes are disabled in 0.13.0, so this should say 0.13.1 for clarity?

Comment (instagibbs) [inline doc/bips.md]: well, they are active on testnet.

File: doc/bips.md [hunk 2]
```diff
@@ -26,4 +26,9 @@ BIPs that are implemented by Bitcoin Core (up-to-date up to **v0.13.0**):
...
+* [`BIP 145`](https://github.com/bitcoin/bips/blob/master/bip-0145.mediawiki): getblocktemplate updates for Segregated Witness as of **v0.13.0** ([PR 7910](https://github.com/bitcoin/bitcoin/pull/7910)).
```
Comment (MarcoFalke) [inline doc/bips.md]: Nit: #7910 was mostly maintained to preserve history. I think you might want to change to #8149 for the rebased version.

**Comment by btcdrak** [comment]
utACK

**Comment by btcdrak** [comment]
@fanquake Could you also update https://github.com/bitcoin-core/bitcoincore.org/blob/gh-pages/_includes/pages/bips-list.md

File: doc/bips.md (hunk 2 above)
Comment (fanquake) [inline doc/bips.md]: Updated to use #8149

**Comment by MarcoFalke** [comment]
ACK ef28d8a

**Review by jonasschnelli — APPROVED** [review]
ACK ef28d8a899aeb4487ef7fbfbfca9c0f2b60bdaf6

**Pull request merged**